cmd/main.go

```diff
@@ -145,6 +149,24 @@ func run(conf *config.Config, choice string) {
 	}
 }
 
+func checkVersion(){
+	var vers string
+	vers = "https://api.github.com/repos/policygenius/monday/releases/latest"
+
+	resp, err := http.Get(vers)
+	if err != nil{
+		fmt.Printf("Internal error on startup, could not check version: %v\n", err)
+		return
+	}
+	dat, _ := ioutil.ReadAll(resp.Body)
+	githubResp := &GithubAPIResponse{}
+	json.Unmarshal(dat, githubResp)
+	if githubResp.TagName != Version {
+		fmt.Printf("🐢 There is a new version of bifrost availabe\n🐢 run ./uninstall --upgrade to get newest version \n")
```

Comment:
Should this be `./install --upgrade` instead?

Reply:
yes it should be!